### ISSUE TITLE
All my images are considered invalid

### ISSUE THREAD

**Issue by Halogenes** [elisemercury/Duplicate-Image-Finder#63]
I try to compare 133 .png pictures but they are all considered invalid by the script.

I don't understand where this can come from because the script was working with these same pictures before I downloaded the latest version (today).

Moreover I can open them with Windows.

Thanks for your work anyway!

```
search = difPy.dif(r"D:\Fichiers conservés\PHOTOS\PNG\1080x1920")

print(search.stats)
```

`{'directory': ['D:\\Fichiers conservés\\PHOTOS\\PNG\\1080x1920'], 'duration': {'start_date': '2023-02-26', 'start_time': '13:21:26', 'end_date': '2023-02-26', 'end_time': '13:21:34', 'seconds_elapsed': 7.862}, 'fast_search': True, 'recursive': True, 'match_mse': 200, 'files_searched': 0, 'matches_found': 0, 'invalid_files': 133}`

**Comment by Halogenes** [comment]
J'ai lancé le même code avec différentes versions et exactement les mêmes images.

v3.0.3 : Ne fonctionne pas
v3.0.2 : Ne fonctionne pas
v3.0.1 : Fonctionne très bien ! (cf ci-dessous)

`{'directory': ('D:\\Fichiers conservés\\PHOTOS\\PNG\\1080x1920',), 'duration': {'start_date': '2023-02-26', 'start_time': '13:30:57', 'end_date': '2023-02-26', 'end_time': '13:31:06', 'seconds_elapsed': 9.0692}, 'fast_search': True, 'recursive': True, 'match_mse': 200, 'files_searched': 133, 'matches_found': 24, 'invalid_files': 0}`

Good luck finding the problem and thanks again for the work :)

(Curious to know what the problem was when you solve it)

**Comment by elisemercury** [comment]
Hi Halogenes,

Thanks for opening the issue! I assume the issue is due to the change from opencv to Pillow (as Pillow is faster and more lightweight). It is likely, that Pillow decodes the images differently and throws an error for certain types of images. If possible, could you please share one (or more) of the images with me, so that I can test what the issue is related to? That would be of great help. You can share the image with me via elisejlandman@hotmail.com. I will work on fixing the issue ASAP!

Thank you and best regards,
Elise

**Comment by Halogenes** [comment]
Sorry for my late reply.

I tried to run exactly the same code, with the same photos using the latest version available (v 3.0.7) and I'm pleased to say that everything works perfectly! 

`{'directory': ['D:\\Fichiers conservés\\PHOTOS\\PNG\\1080x1920'], 'duration': {'start_date': '2023-03-09', 'start_time': '10:45:55', 'end_date': '2023-03-09', 'end_time': '10:46:10', 'seconds_elapsed': 14.6431}, 'fast_search': True, 'recursive': True, 'match_mse': 200, 'files_searched': 133, 'matches_found': 24, 'invalid_files': {'count': 0}`

Congratulations for your work :)
